### ISSUE TITLE
Comment preview results in comments at time of preview being stored, even after submission of the comment

### ISSUE THREAD

**Issue by malachig** [genome/civic-client#452]
- If I write a comment, then submit it, everything works nicely.
- If I write a comment, then preview it, then submit it, the comment is reproduced in a new comment below.  If you don't notice that your comment is submitted it you can accidentally submit a duplicate comment.
- Furthermore, if you write a comment, preview it, return to the edit mode and write more, then submit, the duplicate comment below only shows what you had done up until when you hit the preview button.

Basically, it seems like when you hit the preview button, the contents are cached, and then on submission of the comment, this cache persists in a new comment box.  This may cause confusion on the part of the user.

**Comment by malachig** [comment]
Thanks @jmcmichael !
